universe/Effect.h

```diff
@@ -1123,7 +1123,7 @@ class FO_COMMON_API GenerateSitRepMessage : public EffectBase {
     const std::string& Icon() const
     { return m_icon; }
 
-    const auto& MessageParameters() const
+    auto MessageParameters() -> const std::vector<std::pair<std::string, ValueRef::ValueRefBase<std::string>*>>&
     { return m_message_parameters; }
```

Comment:
The bots are doing more than validating C++ standard versions... A lot of the issues in this pull request / branch have been, I think based on googling the issues or error messages, bugs or missing features in GCC, not standard mismatches. It also checks other inconsistencies like what needs API specifications.